build.gradle

```diff
@@ -4,7 +4,7 @@ import org.apache.tools.ant.taskdefs.condition.Os
 // classpath
 buildscript {
     repositories {
-        maven { url 'https://plugins.gradle.org/m2/' }
+        maven {url "${localNexusLocation}/public" }
```

Comment:
only use Local nexus in CI builds:

eg.:

        if (project.ext.isCi) {
            maven {
                url "${localNexusLocation}/public"
            }

Reply:
right, otherwise you can't build when not on VPN... fixing this, thx.

Reply:
@michalkurka should be fixed now, I ran 2 clean builds locally after cleaning gradle cache: once in CI mode and with VPN, one in dev mode and without VPN.